kurtosis-devnet/interop.yaml

```diff
@@ -173,6 +173,8 @@ ethereum_package:
   participants:
     - el_type: geth
       cl_type: teku
+      cl_extra_env_vars:
```

Comment:
those seem unrelated?

Reply:
ok, I still think we should exclude it from here.

We should:
- run some experiments to try and understand that failure mode better
- if that fix is still needed, then it should go directly into ethereum-package